.gitignore

```diff
@@ -23,6 +23,6 @@ build/Release
 # Deployed apps should consider commenting this line out:
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules
-dist
```

Comment:
@KyleAMathews I agree; that was leftover from the original PR. I didn't alter it on the off chance that I was missing something but since you've confirmed the compiled code shouldn't be there, I'll update the PR.